kratos/python/add_logger_to_python.cpp

```diff
@@ -15,23 +15,35 @@
 
 // Project includes
 #include "includes/define_python.h"
+#include "includes/data_communicator.h"
 #include "input_output/logger.h"
 
 
-
 namespace Kratos {
 namespace Python {
 
-    namespace py = pybind11;
+namespace py = pybind11;
+
+const DataCommunicator& getDataCommunicator(pybind11::kwargs kwargs) {
+    if (kwargs.contains("data_communicator")) {
```

Comment:
this is an implementation-detail right?
We don't have to pass additional stuff from python right?

Reply:
It is an (optional) keyword argument. If you provide one with `logger.PrintSomething("Message", data_communicator=my_comm)`, it will be used instead of the default one.